manifests/params.pp

```diff
@@ -54,6 +54,7 @@
   $major_release = regsubst($::operatingsystemrelease,'^(\d+)\.\d+$','\1')
   $cvmfs_version          = hiera('cvmfsversion','present')
   $cvmfs_yum              = hiera('cvmfs_yum',"http://cern.ch/cvmrepo/yum/cvmfs/EL/${major_release}/${::architecture}")
+  $cvmfs_yum_config       = hiera('cvmfs_yum',"http://cern.ch/cvmrepo/yum/cvmfs-config/EL/${major_release}/${::architecture}")
```

Comment:
This should be just 
```puppet
$cvmfs_yum_config = "http://cern.ch/cvmrepo/yum/cvmfs-config/EL/${major_release}/${::architecture}"
```

as before no new heira calls that I then need to deprecate.  (I'll do that soon.)